INSTALL

```diff
@@ -4,6 +4,10 @@ Meson build system
 glmark2 uses the meson build system for the most common build flavors (X11,
 Wayland, DRM).
 
+Linux dependencies:
+
+$ sudo apt install meson cmake libjpeg-dev libpng-dev libdrm-dev libgbm-dev libudev-dev libwayland-dev wayland-protocols
```

Comment:
Thing is, these are specific to Debian, and not only that but they seem to target the Wayland back-end. 

Rather than specifying that, maybe a better approach would be to just list the package names required for various back-ends -- platforms? Even so, on Arch probably those are named differently, and probably on Fedora as well, but I guess it won't hurt having these here.

As glmark2 is in Debian you might want to cherry-pick the package dependencies, from each platform. 

- X11 depends X, Y, Z
- DRM depends on ...
- Wayland depends on ...